packages/core/test/Cardano/util/estimateStakePoolAPY.test.ts

```diff
@@ -3,32 +3,32 @@ import { Cardano } from '../../../src';
 describe('estimateStakePoolAPY', () => {
   const rewards = {
     epochLength: 432_000_000,
-    memberROI: Cardano.Percent(0.000_68)
+    memberAPY: Cardano.Percent(0.000_68)
```

Comment:
Seems like this is just ROI of a single epoch, which is different from APY (annualized and with compounding). Please review name changes in this commit and make sure they're accurate.

Reply:
As proposed in slack, I also changed the per epoch field from ROI to APY, now it is annualized as well.

Reply:
I think ROI is more appropriate in context of Stake Pool epoch rewards. APY of that epoch can be easily computed by the client if he needs it, as demonstrated by this test.

Reply:
What about providing both?

Reply:
Seems redundant to me, as APY can be computed with `computeStakePoolAPY` util. Thoughts @rhyslbw ?

Reply:
I can't find `computeStakePoolAPY`; maybe was you meaning `estimateStakePoolAPY`?

Reply:
4513ebfda should solve this.
I had to force push to change the commit message too